metrics_metadata.go

```diff
@@ -0,0 +1,292 @@
+package signalfx
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/signalfx/signalfx-go/metrics_metadata"
+)
+
+// DimensionAPIURL is the base URL for interacting with dimensions.
+const DimensionAPIURL = "/v2/dimension"
+
+// MetricAPIURL is the base URL for interacting with dimensions.
+const MetricAPIURL = "/v2/metric"
+
+// MetricTimeSeriesAPIURL is the base URL for interacting with dimensions.
+const MetricTimeSeriesAPIURL = "/v2/metrictimeseries"
+
+// TagAPIURL is the base URL for interacting with dimensions.
+const TagAPIURL = "/v2/tag"
+
+// // CreateDetector creates a detector.
+// func (c *Client) CreateDetector(detectorRequest *detector.CreateUpdateDetectorRequest) (*detector.Detector, error) {
```

Comment:
Seems like the commented out stuff can be removed.

Reply:
Yeaaaah, some lazy C&P here. :)